models/taskRequests.js

```diff
@@ -187,7 +187,7 @@ const fetchTaskRequestById = async (taskRequestId) => {
   };
 };
 
-const createRequest = async (data, authenticatedUsername) => {
+const createRequest = async (data, authenticatedUserId) => {
```

Comment:
What is authenticatedUserId, can it be userId

Reply:
userId -  The user to whom the task request is being created.
authenticatedUserId - The ID of the authenticated user performing the creation. 

Currently we don't have option to create a task request for others but in future there might be. 

Also added jsdoc for the same. Please check

Reply:
Then can we change the name to something else, please its confusing

Reply:
Can you suggest a better name? 

My reasoning for naming it as authenticated user id is because it's the user id which we get after authenticating the jwt token sent in cookies.

Reply:
Can we call createdBy or createdByUserId, creatorUserId, or assignedByUserId , assignedBy, something
authenticatedUserId don't give any meaning

Reply:
author or authorId

Reply:
Changed it to author user Id.